tests/models/mamba/test_modeling_mamba.py

```diff
@@ -460,18 +460,19 @@ def test_simple_generate(self, device):
         self.assertEqual(output_sentence, "Hey how are you doing?\n\nI'm so glad you're here.")
 
         with torch.no_grad():
+            model.float()
```

Comment:
```suggestion
```
Not needed if you change the model loading above to the default `torch.float32`.